classes/Admin/Templates.php

```diff
@@ -562,7 +562,7 @@ public static function conditions_editor() {
 					#>
 				</div>
 				<div class="no-facet-groups">
-					<label for="pum-first-condition"><?php _e( 'Choose a condition to get started.', 'popup-maker' ); ?></label>
+					<label for="pum-first-condition"><?php _e( 'Choose a condition to target where to display your popup.', 'popup-maker' ); ?></label>
```

Comment:
Keep in mind not all conditions will be content. IE Where to display your popup doesn't apply when talking about user or browser based conditions.

Maybe limit who will see this popup?